.codeclimate.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+checks:
+  file-lines:
+    enabled: true
+    config:
+      threshold: 300
+  method-lines:
+    enabled: false
+  method-count:
+    enabled: false
+  method-complexity:
+    enabled: false
+  file-lines:
+    enabled: false
```

Comment:
file-lines is disabled here and enabled on line 3... is that right? seems like it should only be one or the other?